### PR TITLE
DYN-5194 add http protocol to notification link in case it is missing

### PR DESCRIPTION
**Pull request by Enzo707**
Ref.: [[DYN-5194]](https://jira.autodesk.com/browse/DYN-5194)

This PR is for getting rid of app chooser when clicking the notification link. We included a url validation adding the protocol in case it's missing.

**Review**
@QilongTang 
@avidit

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -31,6 +31,16 @@ function App() {
     }
   },[]);
 
+  const validateNotificationUrl = (url) => {
+    let parseURL = url;
+
+    if (!url.includes("//")){
+      parseURL = "http://" + url;
+    }
```
Comment (QilongTang) [inline src/App.js:39]: Looks like a generic problem in javascript from the discussion in https://stackoverflow.com/questions/3543187/prepending-http-to-a-url-that-doesnt-already-contain-http.

**Review by QilongTang — APPROVED** [review]

File: src/App.js (hunk 1 above)
Comment (mjkkirschner) [reply src/App.js:39]: shouldn't this be https? We really shouldn't be using http anywhere.

Comment (Enzo707) [reply src/App.js:39]: Do you prefer a regex allowing both`http` and `https`?

Comment (QilongTang) [reply src/App.js:39]: I think links from our team are usually safe to just bind `https`

**Review by QilongTang — APPROVED** [review]

**Pull request merged**